app/js/services.js

```diff
@@ -0,0 +1,9 @@
+var conceptDirServices = angular.module('conceptDirServices', ['ngResource']);
```

Comment:
Where does Dir come from? It would be clearer for me to name it conceptDictServices.